File: java program/basic/swap.java
class swap
{
    public static void main(String[] args)
    {
        int num1=12, num2=32;
        System.out.println("before swap num1 value: "+ num1);
        System.out.println("before swap num2 value: "+ num2);
        int temp=num1;
        num1=num2;
        num2=temp;
        System.out.println("After swap num1 value: "+ num1);
        System.out.println("After swap num2 value: "+ num2);
    }
}

File: c++ programe/basic/swap.cpp
#include<iostream>
using namespace std;
int main()
{
    int num1 = 11, num2 = 23;
    cout<<"num1 : "<<num1<< endl;
    cout<<"num2 : "<<num2<< endl;
    int temp = num1;
    num1 = num2;
    num2 = temp;
    cout<<endl;
    cout<<"num1 : "<<num1<< endl;
    cout<<"num2 : "<<num2;
    return 0;
}